frontend/shared/src/utils/string.utils.ts

```diff
@@ -26,3 +26,16 @@ export const getBooleanValueFromString = (value?: string): boolean | null => {
 
 export const isEmpty = (value?: string): boolean =>
   typeof value === 'string' ? value?.trim().length === 0 : Boolean(!value);
+
+export const getNumberValue = (s: string): number =>
+  Number(s?.toString().replace(/,/, '.'));
+
+export const stringFloatToFixed2 = (value: string): string =>
```

Comment:
What does 2 refer two in the function name?

Reply:
2 numbers after the . If you find other good name, welcome:)